README.md

```diff
@@ -54,7 +54,7 @@ data:
   PRIVATE_KEY: <encrypted-private-key>
 kind: Secret
 metadata:
-  name: coralogix-keys
+  name: integrations-privatekey
```

Comment:
in logs and metrics its correct that the secret is called `integrations-privatekey`, but in otel integration its called 'coralogix-keys' so it can be confusing if we dont see the otel-agent readme. 
@povilasv I think we should change it in otel so it will match the secret name of the other integrations.

Reply:
@amit-mazor already talked with @oded-dd , after this RP is merged I will change to coralogix-keys in the fluentd/bit integrations as well.